components/zoom_admin/zoom_admin.app.mjs

```diff
@@ -0,0 +1,374 @@
+/* eslint-disable camelcase */
+import axios from "axios";
+import get from "lodash/get.js";
+import sortBy from "lodash/sortBy.js";
+import isArray from "lodash/isArray.js";
+import isEmpty from "lodash/isEmpty.js";
+import isString from "lodash/isString.js";
+import zoomCountries from "./zoom_countries.mjs";
+
+export default {
+  type: "app",
+  app: "zoom_admin",
+  propDefinitions: {
+    cloudRecording: {
+      type: "string",
+      label: "Recording",
+      description: "The ID of the cloud recording. Please use a valid object with `meetingId` and `value` with \"Structured Mode\" disabled. `value` means the recording ID",
+      async options({
+        page,
+        prevContext,
+      }) {
+        const data = await this.listUserRecordings(
+          prevContext.nextPageToken,
+          page + 1,
+        );
+
+        if (!data.meetings) {
+          return [];
+        }
+        const options = [];
+        data.meetings.forEach((meeting) => {
+          meeting.recording_files.forEach((recording) => {
+            const label = `${meeting.topic} - (Format: ${recording.file_type}) (From ${recording.recording_start} to ${recording.recording_end})`;
+            options.push({
+              label,
+              value: {
+                meetingId: meeting.id,
+                label,
+                value: recording.id,
+              },
+            });
+          });
+        });
+
+        return {
+          options,
+          context: {
+            nextPageToken: data.next_page_token,
+          },
+        };
+      },
+    },
+    meeting: {
+      type: "string",
+      label: "Meeting",
+      description: "The meeting ID",
+      async options({
+        prevContext,
+        page,
+      }) {
+        if (!prevContext.nextPageToken && page > 0) {
+          return [];
+        }
+        const data = await this.listMeetings(prevContext.nextPageToken);
+        return {
+          options: data?.meetings.map((meeting) => ({
+            label: meeting.topic,
+            value: {
+              label: meeting.topic,
```

Comment:
I think is not necessary to assign the label again here for `$summary` porpuses since the summary can be more generic without the need of specific `ids` or `labels`. According to the [docs](https://pipedream.com/docs/components/guidelines/#use-summary-to-summarize-what-happened) the structure should be simple right? the same commet goes for the rest of `async options` with the same aproach where you are including the `label` in the `value`.

Reply:
I disagree with it. According to the docs: "_Whenever possible, use names and titles instead of IDs_". The better way to do it is sending an object with the value and the name. Please check this [merged PR](https://github.com/PipedreamHQ/pipedream/pull/1762) using this same approach. If you still disagreeing, please let me know. Maybe we can move this discussion to Slack in order to found the best approach :)

Reply:
Ok let me ask this on Slack because if that is true, it should be like an standard for the rest of PRs rigth?

Reply:
Yeah, let's try to get a more detailed answer from @dannyroosevelt  or @dylburger and so we can go ahead with the result of our slack thread. :)

Reply:
In my view, it should be optional, the dev could have the possibility to follow a more detailed approach like this, or use simple messages.